web/app/Helpers/LegacyTools.php

```diff
@@ -116,6 +104,26 @@ public static function generateScreenVars()
             }
         }
 
+        return $site;
+    }
+
+    /**
+     * Bootstrap a runData instance and generate the needed vars to give to a blade template.
+     * @return array|string
+     * @throws \Wikidot\Utils\ProcessException
+     */
+    public static function generateScreenVars()
+    {
+        /**
+         * Create a RunData instance.
+         */
+        $runData = new RunData();
+        $runData->init();
+        Ozone::setRunData($runData);
+        Log::debug('runData object created and initialized in LegacyTools::generateScreenVars()');
+
+        $site = static::getCurrentSite();
```

Comment:
Is there a reason you're doing `static::xxx` instead of `self::x`? Is this something prettier is doing?

Reply:
Uhhhhh I just do `static` I don't know if there is actually a difference